.idea/checkstyle-idea.xml

```diff
@@ -3,11 +3,11 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
-        <entry key="active-configuration-0" value="PROJECT_RELATIVE:C:/private/geocaching/github/cgeo/checkstyle.xml:cgeo" />
+        <entry key="active-configuration-0" value="PROJECT_RELATIVE:$PROJECT_DIR$/checkstyle.xml:cgeo" />
```

Comment:
You accidentally checked in local changes to your checkstyle-config

Reply:
This is your private path to the repository, isn't it? I guess we should all add this file to our local ignore list (https://stackoverflow.com/a/23438899/13513639):
```
git update-index --assume-unchanged ".idea/checkstyle-idea.xml"
```


![grafik](https://user-images.githubusercontent.com/64581222/168092613-aabfedd3-309f-4ffd-a8d4-ba4f73621461.png)

Reply:
whoops...